slippyMap/resources/LeafletMap.js

```diff
@@ -13,7 +13,7 @@ export default class LeafletMap {
 
```

Comment:
Imho some more explanatory comments would help to read the code because many things for initialising, setting, re-setting, etc. are done here, it's sometimes a bit hard to follow.